tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1787,8 +1789,10 @@ The following pattern types are available:
             "Sword", "Splash potion", "Fist":
                 print("Yep, you've taken damage")
 
-Pattern guards
-""""""""""""""
+.. _doc_gdscript_basics_pattern_guards:
+
+Pattern guard expressions
+"""""""""""""""""""""""""
```

Comment:
I'll defer to you on the terminology since you wrote this section originally (https://github.com/godotengine/godot-docs/pull/8094) and reviewed the implementation (https://github.com/godotengine/godot/pull/80085).

I think replacing "guard expression" with "pattern guard" in all instances is the right way to go. Not sure if that's what you were suggesting, or only the narrow case of that code block.

---
Looking at the source and user-visible error messages again, it seems like "pattern guard" is the official term for the expression.

https://github.com/godotengine/godot/blob/5ca419e32c58e671284b3b7f91257179657d9114/modules/gdscript/gdscript_parser.cpp#L2251
https://github.com/godotengine/godot/blob/5ca419e32c58e671284b3b7f91257179657d9114/modules/gdscript/tests/scripts/parser/features/allowed_keywords_as_identifiers.gd#L18

"Guard expression" and "pattern guard" both refer to the expression after "when", but "guard expression" doesn't appear anywhere in the source.